added solutions

engineer027 opened the pull request.

engineer027 committed:
added solutions

engineer027 committed:
added DeleteDriverController

engineer027 committed:
added:
DisplayAllManufacturersController;
DeleteManufacturerController;
manufacturers.jsp.
updated:
index.jsp;
web.xml.

engineer027 committed:
added:
DisplayAllCarsController;
DeleteCarController;
manufacturers.jsp.
updated:
index.jsp;
web.xml.

RuslanIbIsh commented on src/main/java/mate/academy/controllers/AddDriverToCarController.java:
```diff
@@ -0,0 +1,37 @@
+
+public class AddDriverToCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+    CarService carService = (CarService) injector.getInstance(CarService.class);
```
сделай его private final - было в чеклисте

RuslanIbIsh commented on src/main/webapp/WEB-INF/views/drivers/drivers.jsp:
```diff
@@ -0,0 +1,34 @@
+            </td>
+        </tr>
+    </c:forEach>
+
```
лишняя строка)))

engineer027 committed:
added access modifiers

engineer027 committed:
fix table cars.jsp

engineer027 committed:
fix travis

KseniiaMakarova commented on src/main/java/mate/academy/controllers/CreateCarController.java:
```diff
@@ -0,0 +1,37 @@
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String model = req.getParameter("model");
+        Long manufacturerId = Long.valueOf(req.getParameter("manufacturerId"));
```
```suggestion
        Long manufacturerId = Long.valueOf(req.getParameter("manufacturer_id"));
```
let's use the same approach everywhere for request parameters

KseniiaMakarova commented on src/main/java/mate/academy/controllers/CreateDriverController.java:
```diff
@@ -0,0 +1,33 @@
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String name = req.getParameter("name");
+        String licenceNumber = req.getParameter("licenceNumber");
```
```suggestion
        String licenceNumber = req.getParameter("licence_number");
```

KseniiaMakarova commented on src/main/webapp/WEB-INF/views/cars/create.jsp:
```diff
@@ -0,0 +1,19 @@
+    <input type="text" name="model" required><br>
+    Please provide manufacturer id<br>
+    <input type="number" name="manufacturerId" required><br>
+    <input type="reset">
```
`<input type="reset">` what is this for? and how does it look?

KseniiaMakarova commented on src/main/webapp/WEB-INF/views/cars/drivers/add.jsp:
```diff
@@ -0,0 +1,19 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Create car</title>
```
```suggestion
    <title>Add driver to car</title>
```

KseniiaMakarova reviewed (approved):
there only a few comments above

engineer027 committed:
fix:
cars\add.jsp;
cars\create.jsp;
drivers\create.jsp;
class CreateCarController;
class CreateDriverController.
